components/library/gnutls-3/Makefile

```diff
@@ -25,13 +25,13 @@
 # Copyright (c) 2022, Andreas Wacknitz
 #
 
-BUILD_BITS=		64_and_32
+BUILD_BITS=		64
 include ../../../make-rules/shared-macros.mk
```

Comment:
When removing 32 bit we'll need to check and change all dependent packages (recursively):
pkg search -r -H -o pkg.name "depend:require:library/gnutls-3" | sort -u
application/cluster/pacemaker
crypto/gnupg
database/yaz
desktop/nntp/pan
desktop/remote-desktop/rdesktop
desktop/remote-desktop/tigervnc
diagnostic/wireshark/wireshark-common
editor/gnu-emacs/gnu-emacs-gtk
editor/gnu-emacs/gnu-emacs-x11
gnome/vino
library/desktop/gtk-vnc
library/desktop/vte-291
library/glib-networking
library/libmicrohttpd
library/libzip
library/print/cups-libs
library/samba/libsmbclient
library/xmlsec
mail/claws-mail
mail/mailutils
network/chat/weechat
print/qpdf
service/network/samba
system/rsyslog
web/aria2
x11/server/xvnc

As you can see there are some 64 bit-only packages already but some deliver also 32 bit.
Alas it looks like we cannot drop 32 bit gnutls-3 at the moment...

Reply:
I had to change autogen to include 32-bit libopts so this can compile. #9300